browser/js/toggl-module.js

```diff
@@ -9,190 +9,190 @@ require('moment-duration-format');
+            var tokens = [];
+            for (var i = 0; i < localStorage.length; ++i) {
+                const key = localStorage.key(i);
+                tokens.push(localStorage.getItem(key));
+            }
+            return tokens;
+        };
+
+        async.eachSeries(apiTokens(), function(apiToken, callback) {
+            new TogglClient({apiToken: apiToken})
       .getUserData({with_related_data: true}, function(err, userData) {
-        appendUserDataContainer(userData);
-        callback(err, userData);
+          appendUserDataContainer(userData);
+          callback(err, userData);
       });
-    }, function(err) {
-      if (err) {
-        console.error(err);
-      }
-      saveScreenShot();
-    });
-
-    $interval(function() {
-      async.eachSeries(apiTokens(), function(apiToken, callback) {
-        new TogglClient({apiToken: apiToken})
+        }, function(err) {
+            if (err) {
+                console.error(err);
+            }
+            saveScreenShot();
+        });
+
+        $interval(function() {
+            async.eachSeries(apiTokens(), function(apiToken, callback) {
+                new TogglClient({apiToken: apiToken})
         .getUserData({with_related_data: true}, function(err, userData) {
-          if(err) {
-            callback(err);
-            return;
-          }
-          updateUserDataContainer(userData);
-          callback(null, userData);
-        })
-      }, function(err){
-        if(err) {
-          console.error(err);
-        }
-        saveScreenShot();
+            if (err) {
+                callback(err);
+                return;
+            }
+            updateUserDataContainer(userData);
+            callback(null, userData);
+        });
+            }, function(err) {
+                if (err) {
+                    console.error(err);
+                }
+                saveScreenShot();
+            });
+
+        }, 10 * 1000);
+
+        $interval(function() {
+            $scope.userDataContainer
+            .users
+            .filter(function(u) {return u.doing_now;})
+      .forEach(function(u) {
+          var range = new Date().getTime() - Date.parse(u.current_time_entry.start);
+          u.current_time_entry.duration_output =  moment.duration(range, 'ms').format('h [hrs], m [min], s [sec]');
       });
+        }, 1000);
+    })();
+
+    var appendUserDataContainer = function(userData) {
+        var lastTimeEntry = userData.time_entries[userData.time_entries.length - 1];
+        var currentTimeEntry = null;
+        if (lastTimeEntry.duration < 0) {
+            currentTimeEntry = lastTimeEntry;
+            var range = new Date().getTime() - Date.parse(currentTimeEntry.start);
+            currentTimeEntry.duration_output = moment.duration(range, 'ms').format('h [hrs], m [min], s [sec]');
+        }
+        userData.current_time_entry = currentTimeEntry;
+        userData.doing_now = currentTimeEntry != null;
```

Comment:
Identifier 'doing_now' is not in camel case.<br>Use '!==' to compare with 'null'.